modules/distribution/src/assembly/bin.xml

```diff
@@ -243,23 +203,54 @@
         </fileSet>
         <fileSet>
             <directory>
-                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/security/
+                ../p2-profile-gen/target/wso2carbon-core-${carbon.kernel.version}/repository/conf/security
             </directory>
-            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/security/</outputDirectory>
+            <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/security
+            </outputDirectory>
             <includes>
+                <include>**/sso-sp-config.properties</include>
+                <include>**/entitlement.properties</include>
                 <include>**/trusted-idp-config.xml</include>
-                <!--============ Added with ES Features ===============-->
                 <include>**/cipher-text.properties</include>
                 <include>**/cipher-tool.properties</include>
-                <include>**/cipher-standalone-config.properties</include>
-                <!--============ Added with ES Features ===============-->
             </includes>
             <excludes>
                 <exclude>authenticators.xml</exclude>
             </excludes>
         </fileSet>
 
+        <!-- <fileSet>
+             <directory>
+                 src/repository/conf/identity/
+             </directory>
+             <outputDirectory>${pom.artifactId}-${pom.version}/repository/conf/identity/</outputDirectory>
+             <includes>
+                 <include>**/**</include>
+             </includes>
+         </fileSet>-->
```

Comment:
Already fixed in commit https://github.com/wso2-incubator/product-iot-server/commit/febdc36cba8b94c3ebcaba27947893ebb2865513 as per the PrabathA's comment.